packages/admin/admin-stories/src/docs/components/DataGrid/DataGrid.stories.tsx

```diff
@@ -0,0 +1,186 @@
+import { gql, useQuery } from "@apollo/client";
+import {
+    CrudContextMenu,
+    GridFilterButton,
+    Toolbar,
+    ToolbarFillSpace,
+    ToolbarItem,
+    useBufferedRowCount,
+    useDataGridRemote,
+    usePersistentColumnState,
+} from "@comet/admin";
+import Box from "@mui/material/Box";
+import { DataGrid, GridColDef } from "@mui/x-data-grid";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+import { apolloStoryDecorator } from "../../../apollo-story.decorator";
+import { storyRouterDecorator } from "../../../story-router.decorator";
+
+const exampleRows = [
+    { id: 1, lastName: "Snow", firstName: "Jon" },
+    { id: 2, lastName: "Lannister", firstName: "Cersei" },
+    { id: 3, lastName: "Lannister", firstName: "Jaime" },
+    { id: 4, lastName: "Stark", firstName: "Arya" },
+    { id: 5, lastName: "Targaryen", firstName: "Daenerys" },
+    { id: 6, lastName: "Melisandre", firstName: null },
+    { id: 7, lastName: "Clifford", firstName: "Ferrara" },
+    { id: 8, lastName: "Frances", firstName: "Rossini" },
+    { id: 9, lastName: "Roxie", firstName: "Harvey" },
+];
+const exampleColumns: GridColDef[] = [
+    {
+        field: "firstName",
+        headerName: "First name",
+    },
+    {
+        field: "lastName",
+        headerName: "Last name",
+    },
+];
+
+storiesOf("stories/components/DataGrid", module)
+    .addDecorator(storyRouterDecorator())
+    .addDecorator(apolloStoryDecorator(`https://api.spacex.land/graphql/`))
+    .add("useDataGridRemote", () => {
+        const columns: GridColDef[] = [
+            {
+                field: "mission_name",
+                headerName: "Mission Name",
+            },
+            {
+                field: "launch_date_local",
+                headerName: "Launch Date",
+                type: "dateTime",
+            },
+        ];
+
+        const dataGridProps = useDataGridRemote();
+
+        const { data, loading } = useQuery(
```

Comment:
MUI DataGrid also has an error prop, would be nice when we could forward the apollo error directly to the DataGrid and provide best practice in stories.